include/flexi_cfg/config/parser-internal.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <tao/pegtl/config.hpp>
+
+#include "flexi_cfg/config/trace-internal.h"
+
+namespace flexi_cfg::config::internal {
+namespace peg = TAO_PEGTL_NAMESPACE;
+
+template <typename Rule, template <typename...> class Action = peg::nothing,
+          template <typename...> class Control = peg::normal, typename ParseInput, typename... States>
+auto parseCore(ParseInput&& input, States&&... output) -> bool {
```

Comment:
this file is to wrap all pegtl parse calls to turn on the grammar tracer everywhere from a single define